### PR TITLE
fix(select component): fixed select body position, show select compon…

### LINKED ISSUE #265: select doesn't work on mobile and desktop
# Summary

The component Select doesn't work properly on mobile, tested on Opera mini and Chrome. It's realy hard to get it open

**Note**: seems that the floatui struggles to work properly, there's like a layout shifting

image

### PR DESCRIPTION
**Pull request by a-lukasiewicz**
fixed select body position, show select component properly

fix #265

# What is it?

- [ ] Feature / enhancement
- [x] Bug
- [ ] Docs / tests

# Description

Select component was not working properly. After applying some styles and removing function to calculate proper position everything is working fine.

# Use cases and why

- 1. One use case
- 2. Another use case

# Screenshots/Demo

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by a-lukasiewicz** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/website/src/routes/docs/daisy/select/index.tsx [hunk 1]
```diff
@@ -1,44 +1,50 @@
...
-import { Select } from '@qwik-ui/headless';
+import {
+  Root,
+  Trigger,
+  Value,
+  ListBox,
+  Group,
+  Label,
+  Option,
+} from '@qwik-ui/headless';
```
Comment (gioboa) [inline apps/website/src/routes/docs/daisy/select/index.tsx:10]:
```suggestion
import { Select } from '@qwik-ui/headless';
```

File: packages/headless/src/index.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export * from './components/tabs/tabs';
 export * from './components/tooltip/tooltip';
 export * as Checkbox from './components/checkbox/checkbox';
 export * as CheckboxProps from './components/checkbox/checkbox';
-export * as Select from './components/select/select';
+export * from './components/select/select';
 export * from './components/slider';
```
Comment (gioboa) [inline packages/headless/src/index.ts:19]:
```suggestion
export * as Select from './components/select/select';
export * from './components/slider';
```

**Comment by gioboa** [comment]
@a-lukasiewicz I had few errors so I fixed them. Anyway we are using `@floating-ui/dom` to detect the right position for the select options. e.g. If the select is at the bottom of the page, the options need to appear on top.
with this PR we are losing this behaviour.

**Comment by a-lukasiewicz** [comment]
@gioboa Finally I made that, now it's ready for review

As on a video I placed select component on a bottom of page and it's working fine 

https://user-images.githubusercontent.com/59919290/232807668-fb28cbe7-88d2-4bbe-bdb8-06e2fb9e916b.mov

**Comment by gioboa** [comment]
Thanks @a-lukasiewicz this PR is really old and `daisy` theme is not available anymore.
I'm so sorry because you've done a great job. You are more than welcome to contribute again on this project.

**Pull request closed without merging**